jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/component/JwalaUi.java

```diff
@@ -319,4 +319,14 @@ public void screenShot(final String imgPath) {
             throw new JwalaUiException("Screen shot failed!", e);
         }
     }
+
+    //the dialog box xpaths have been generalized since different popups have different xpaths
+    public void clickOk(){
+        click(By.xpath("//*[text()='Ok']"));
```

Comment:
Yeah it is case-sensitive but it makes sense since the test just checks for the element actually present in our UI, we don't need to make it case-insensitive 😄